doc/release-notes.md

```diff
@@ -21,78 +21,659 @@ How to Upgrade
+- [`217f3941d`](https://github.com/dashpay/dash/commit/217f3941d) Skip starting of cache populator thread in case we don't have a valid vvec
+- [`679a9895b`](https://github.com/dashpay/dash/commit/679a9895b) Add comments about why it's ok to ignore some failures
+- [`15c34ccbd`](https://github.com/dashpay/dash/commit/15c34ccbd) Implement CQuorum and CQuorumManager
+- [`8e4fe3660`](https://github.com/dashpay/dash/commit/8e4fe3660) [PrivateSend] Fallback to less participants if possible, fix too long timeouts on server side (#2616)
+- [`ee808d819`](https://github.com/dashpay/dash/commit/ee808d819) Add checkbox to show only masternodes the wallet has keys for (#2627)
+- [`fff50af3c`](https://github.com/dashpay/dash/commit/fff50af3c) Revert "Set CLIENT_VERSION_IS_RELEASE to true (#2591)"
+- [`fed4716c0`](https://github.com/dashpay/dash/commit/fed4716c0) Remove support for "0" as an alternative to "" when the default is requested (#2622)
+- [`8b7771a31`](https://github.com/dashpay/dash/commit/8b7771a31) Add some `const`s
+- [`0b1347c0d`](https://github.com/dashpay/dash/commit/0b1347c0d) Pass self-created message to CDKGPendingMessages instead of processing them
+- [`02c7932f4`](https://github.com/dashpay/dash/commit/02c7932f4) Add owner and voting addresses to rpc output, unify it across different methods (#2618)
+- [`c948c0ff3`](https://github.com/dashpay/dash/commit/c948c0ff3) Fix help for optional parameters in "quorum dkgstatus"
+- [`957652bf3`](https://github.com/dashpay/dash/commit/957652bf3) Fix help for "quorum dkgstatus" and remove support for "0" proTxHash
+- [`b7b436b7d`](https://github.com/dashpay/dash/commit/b7b436b7d) Apply review suggestions to rpcquorums.cpp
+- [`3fe991063`](https://github.com/dashpay/dash/commit/3fe991063) Drop unused overload of GetMasternodeQuorums
+- [`5daeedabf`](https://github.com/dashpay/dash/commit/5daeedabf) Batched logger should not break log parsing
+- [`2aed51c55`](https://github.com/dashpay/dash/commit/2aed51c55) Give nodes more time per phase when doing PoSe tests
+- [`5958f8b81`](https://github.com/dashpay/dash/commit/5958f8b81) Remove dkgRndSleepTime from consensus params and make sleeping it non-random
+- [`0dae46c2f`](https://github.com/dashpay/dash/commit/0dae46c2f) Move RandBool() into random.h/cpp
+- [`e1901d24a`](https://github.com/dashpay/dash/commit/e1901d24a) Handle review suggestions
+- [`352edbd33`](https://github.com/dashpay/dash/commit/352edbd33) Introduce SPORK_18_QUORUM_DEBUG_ENABLED to enable/disable LLMQ debug messages
+- [`324406bfe`](https://github.com/dashpay/dash/commit/324406bfe) Implement debugging messages and RPC for LLMQ DKGs
+- [`098b09495`](https://github.com/dashpay/dash/commit/098b09495) Pass scheduler to InitLLMQSystem
+- [`a1f4853d6`](https://github.com/dashpay/dash/commit/a1f4853d6) Use LLMQ DKGs for PoSe testing in DIP3 tests
+- [`6836f8c38`](https://github.com/dashpay/dash/commit/6836f8c38
```

Comment:
Isn't this list normally sorted by activity / number of PRs?

Reply:
No, it _should_ be alphabetically sorted (we messed up in 0.13.0 release but it should be sorted this way in all other releases afaik).

Reply:
:+1: